Fix up PrometheusRules validation (#351)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-rule-in-prometheus.adoc

  groups:
    - name: ./openstack.rules
      rules:
        - alert: Collectd metrics receive rate is zero
          expr: rate(sg_total_collectd_msg_received_count[1m]) == 0 # <1>
EOF
----
<1> To change the rule, edit the value of the `expr` parameter.

> . Validate that the Operator loaded the rules into Prometheus by running `curl` against the default-prometheus-proxy route with basic authentication:

Reviewer: ```suggestion
. To verify that the Operator loaded the rules into Prometheus, run the `curl` command against the default-prometheus-proxy route with basic authentication:
```
